lib/.eslintrc.yaml

```
@@ -57,6 +57,8 @@ rules:
       # disabled with --no-harmony-sharedarraybuffer CLI flag.
     - name: SharedArrayBuffer
       message: "Use `const { SharedArrayBuffer } = globalThis;` instead of the global."
+    - name: StructuredClone
```

The name is case-sensitive. 
```suggestion
    - name: structuredClone
```

Actually, can you put it below `queueMicrotask` in this list as it seems functions are coming after classes in this list (`atob` is below `WebAssembly`).